### PR TITLE
removed Send Feedback from dropdown et al

### PR DESCRIPTION
**Pull request by Israellund**
## Link to Issue
Closes: #8163 

## Description of Changes
-removed Send Feedback from dropdown
-removed any logic corresponding to sending feedback
-removed send feedback modal from codebase

## Test Plan
-hover over help icon in header and confirm 'Send feedback' is no longer shown.

### CODE REVIEW

File: packages/commonwealth/client/scripts/views/modals/feedback_modal.tsx [hunk 1]
```diff
@@ -1,77 +0,0 @@
...
-import type { ValidationStatus } from '../components/component_kit/cw_validation_text';
-import { CWValidationText } from '../components/component_kit/cw_validation_text';
-import { CWButton } from '../components/component_kit/new_designs/CWButton';
-import {
-  CWModalBody,
-  CWModalFooter,
-  CWModalHeader,
-} from '../components/component_kit/new_designs/CWModal';
-
-import '../../../styles/modals/feedback_modal.scss';
```
Comment (masvelio) [inline packages/commonwealth/client/scripts/views/modals/feedback_modal.tsx:15]: please remove scss file

File: packages/commonwealth/server/routing/router.ts [hunk 2]
```diff
@@ -66,7 +66,6 @@ import getUploadSignature from '../routes/getUploadSignature';
 
 import bulkOffchain from '../routes/bulkOffchain';
 import logout from '../routes/logout';
-import sendFeedback from '../routes/sendFeedback';
```
Comment (masvelio) [inline packages/commonwealth/server/routing/router.ts:69]: if you are removing the whole flow, please also remove backend `sendFeedback` file. How about `SLACK_FEEDBACK_WEBHOOK` env var and slack integration?

**Comment by mzparacha** [comment]
Just had a look, I have same comments as Marcin.

**Comment by Israellund** [comment]
@masvelio @mzparacha Just pushed some new changes, let me know what you think. Also reached out to Mark and Rhys to see where the feedback actually went so that I can disable everything completely.

**Review by mzparacha — APPROVED** [review]

**Review by masvelio — APPROVED** [review]

**Pull request merged**